ops/ansible/secrets.yml

```diff
@@ -5,11 +5,38 @@
 
 deploy_discord_webhook_token: !vault |
           $ANSIBLE_VAULT;1.1;AES256
-          34313137643130353633383862633539333461613734366662386363316264353539613232636262
-          3335353236356434646333613266346536336165336437630a663437343063383632303065363764
-          32313033613439393466373031343039666430353862306238383364613438623866346366663434
-          6631393638653334350a376235323533626331343036653030666637373139396430633261306634
-          32356536333463373865383666306533313431303566633065353263643539386263316361356639
-          38326331633063666434333639613563356533386532363565626561323361363562633338353563
-          30663735643232666531313939376633656639636436636565653130616161376334646636363862
-          39623463666663353362
+          36653038373964623266393535303362366235303236396636376131346331363931623463323563
+          3638353566353563666563653531613938616162306364320a666539636263393035326633393739
+          61633137356463613862393637613433613639306661396563386461393036636466333836316637
+          6132393033313364300a323666303061303631666130643966643530383338303464383231653263
+          63616262663932313535333736323663396439346534316339396564303930303961666131383237
+          63333464303862366564623965393266353465303230643138646466376638323565343961383134
+          62646561373163656134343939633161623132623437376639343735303639353532626561653666
+          34333636626363353731
+backup_creds_id: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          61393962313933626630313031363735313062346637306266353133353339333630663139333964
+          3338666631313730356338623766646266656631383936620a623237303764663232653139396565
+          62306664626339306339323730343136396165616130373538333633316331393038343132336264
+          3134633766363937650a386366326138643965336338626163653034316333343866393766633736
+          63656531343237376366323537613336323832623765323938376163376434366138
+backup_creds_key: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          38366334333131336434623562613062383464626561346239313631313166346364363534636231
+          3739323965643933303863383634373266333737343936620a646165383636316462393163653666
+          35303232353630633065373736316433633366336331313464376465386436333264353333393631
+          6465396138323438390a636436303763333633623534343231303239663339303032353131643536
+          34373430306139336266323264306336313339383462366330376665343265336138623834623263
+          3030376337616536616363343664613132666334626232663865
+discord_webhook_url: !vault |
```

Comment:
Can we make the Discord webhook secrets symmetrical? Maybe we could name them `discord_deploy_webhook_url` and `discord_backup_webhook_url`?

`discord_deploy_webhook_url` would replace `deploy_discord_webhook_token`, as per the comment above.